h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> A question is whether we need this information at all. Shouldn't we be able to tell, at the completion callback, which way we've scrolled by which edge the element is aligned with?

Not really. The library is tricky, because it examines all the parents of the given DOM node, and tries to intelligently determine which one[s?] to scroll. So upon the completion of the callback, it's not entirely obvious which coordinates to check, and what to compare them against. We could try to add some intelligence here, but I see no point in doing that, since the library is perfectly aware of what it is doing, we just have to get that information out of it.